definitions/npm/sequelize_v4.x.x/flow_v0.42.x-/test_sequelize.js

```diff
@@ -1,9 +1,9 @@
 // @flow
+const Sequelize = require("sequelize");
+const { Model, Transaction } = Sequelize;
 
-import Sequelize, {Model, Transaction} from "sequelize";
```

Comment:
@pascalduez deleting this line is unacceptable and it should have never passed review.

Reply:
Agree, we should test both commonjs and ES modules when doing those kind of conversions.